src/statecharts/impl.cljc

```diff
@@ -734,11 +734,12 @@
         atomic-nodes (filter #(= (:type %) :atomic) configuration)
         txs (->> atomic-nodes
                  (map #(select-one-tx fsm % state event input-event)))
-        _ (when-not (->> txs
+        _ (when-not ignore-unknown-event?
+            (when-not (->> txs
                          (map first)
                          (some identity))
-            (throw (ex-info (str "fsm " (:id fsm) " got unknown event " (:type event))
-                            {:_state _state})))
+              (throw (ex-info (str "fsm " (:id fsm) " got unknown event " (:type event))
+                              {:_state _state}))))
```

Comment:
Hey @DougHamil thanks for this patch. The change should work, but with all due respect I don't like mixing the major changes with reformatting of code. 

So would you please either:
1. revert all formatting/whitespace changes, or
2. split the whitespace changes (that are not related to this patch) into a separate commit so each commit is atomic itself.

Reply:
Also the CI is not running for this PR. I have updated the master branch to enable github actions on pull requests, so please rebase your branch to make the CI run on it.

Reply:
Whoops, my parinfer got a little aggressive! I've reverted the whitespace changes and rebased on latest master.

Let me know if there are any other adjustments I should make!